zebra/zebra_vty.c

```diff
@@ -151,27 +340,31 @@ static int zebra_static_route_leak(struct vty *vty,
 	memset(&snh_label, 0, sizeof(struct static_nh_label));
 	if (label_str) {
 		if (!mpls_enabled) {
-			vty_out(vty,
-				"%% MPLS not turned on in kernel, ignoring command\n");
+			if (vty)
+				vty_out(vty,
+					"%% MPLS not turned on in kernel, ignoring command\n");
 			return CMD_WARNING_CONFIG_FAILED;
```

Comment:
that event should be logged into a zlog_xxx

Reply:
done